stream-chat-android-ui-utils/src/main/kotlin/io/getstream/chat/android/uiutils/extension/Channel.kt

```diff
@@ -108,38 +108,57 @@ private fun Channel.nameFromMembers(
  *
  * @param context The context to load string resources.
  * @param currentUser The currently logged in user.
+ * @param countCurrentUserAsOnlineMember If `true`, the current user will be counted as an online member.
  * @return The text that represent the member status of the channel.
+ * @param countOtherUsersAsOnlineMembers If `true`, other users will be counted as online members.
  */
+@Suppress("LongParameterList")
 public fun Channel.getMembersStatusText(
     context: Context,
     currentUser: User?,
+    countCurrentUserAsOnlineMember: Boolean = true,
+    countOtherUsersAsOnlineMembers: Boolean = true,
```

Comment:
As I said before, would be better to receive only one instance that wrap both flags to be able to extend it in the future without modifying the public API

Reply:
As agreed in call, we will keep it for now and do a refactor in another opportunity, as this requires more changes than expected due to how `ui-common` and `ui-utils` relate with `ui-component` and `compose` modules.